internal/handler/wallet/add_assets_test.go

```diff
@@ -0,0 +1,91 @@
+package wallet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/massalabs/station-massa-wallet/api/server/models"
+	utils "github.com/massalabs/station-massa-wallet/pkg/assets"
+	"github.com/massalabs/station-massa-wallet/pkg/wallet"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAddAssetHandler_InvalidAddress(t *testing.T) {
+	api, _, _, _, err := MockAPI()
+	assert.NoError(t, err)
```

Comment:
we usually do like this:

```
	api, _, _, _, err := MockAPI()
	assert.NoError(t, err)

	// test empty configuration first.
	t.Run("Get empty list", func(t *testing.T) {
		resp, err := processHTTPRequest(api, "GET", "/api/accounts", "")
		assert.NoError(t, err)
```

to not duplicate the creation of the API with `MockAPI`